cylc/flow/task_job_mgr.py

```diff
@@ -617,40 +617,6 @@ def _create_job_log_path(workflow, itask):
                 exc.filename = target
             raise exc
 
-    @staticmethod
-    def _get_job_scripts(itask, rtconfig):
```

Comment:
This seemed to be a double up of what's already done in the `config.py` .. The only difference is the omission of environment `CYLC_TASK_CYCLE_POINT` for `str(itask.point)` (which isn't available earlier):
```
            comstr = (
                "cylc workflow-state "
                + " --task=" + itask.tdef.workflow_polling_cfg['task']
                + " --point=" + str(itask.point)
            )
```
This doesn't appear to have an impact, however, I could be wrong.. (is the env variable ever not available?)

Reply:
`CYLC_TASK_CYCLE_POINT` should always be available in job execution environments so should be good.

Took a look and I think we are safe to remove this, cheers.